app/snippet.rb

```diff
@@ -0,0 +1,10 @@
+class Snippet
+  def self.generate(html)
+    html
+      .split('</h1>')[1..-1].join # make sure we skip anything before <h1>
+      .gsub(/(\<h.+\>)/, "") # remove headings
```

Comment:
Is this regex aiming to remove `<h*>` tags or the entire header? I think it's excessively gready either way.